dist/amd/hansken-js.js

```diff
@@ -1,4 +1,4 @@
-define('modules/keyManager.js',["exports"], function (_exports) {
```

Comment:
Isn't it uncommon to check in the `dist` folder? At least in Python I think it isn't done typically. Not sure about Javascriptland.

Reply:
Yeah, it surprised me as well, but by checking it in, it can be used directly via npmjs without the need of a build script